itsy.css

```diff
@@ -0,0 +1,87 @@
+body {
+  margin: 0px;
+}
+header {
+  display: flex;
+  background: url(img/lead.jpg);
+  height: 100vh;
+  flex-direction: column;
+  justify-content: space-between;
+
+}
+
+.header-top {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 30px;
+  font-family: 'Amatic SC', 'Heveltica Neue', Arial, sans-serif;
+  font-size: 30px;
+  background: rgba(255,255,255, 0.8);
+
+}
+
+.header__list {
+  list-style: none;
+  float: left;
+  text-transform: uppercase;
+  margin-left: 30px;
+  font-size: 25px;
+}
+
+.header__icon {
+  color: #56566e;
+  font-size: 15px;
+}
+
+.icon__search {
+  color: #e68a8a;
```

Comment:
it looks more as active state of icon. Please change css selector to `.active` and use that instead of `icon__search`